### ISSUE TITLE
Show "Brand" instead of "Name"

### ISSUE THREAD

**Issue by MacG-DD** [fewieden/MMM-Fuel#75]
I use Tankerkönig and it would make more sense to show "Brand" instead of "Name". The brand is provided by the API next to the name and address.

Some gas stations here have only capital letters in the name and others don't have the brand in the name. Anyway, that would give the table a more pleasing content than this mess.

In the name is usually also the address and we actually know our gas stations in the area of our MagicMirror. The brand should be sufficient.

**Comment by leovandavart** [comment]
is there any news yet 

the second gas station has unfortunately the address as name 

I would also like to show the company/brand because that is the correct name the name is HEM
![tanken](https://user-images.githubusercontent.com/51362702/158078024-1d649362-0f8e-4187-9ffa-de4f01aefa96.PNG)

tankerkönig unfortunately has the wrong name

  {
    "id": "e1a15081-24be-9107-e040-0b0a3dfe563c",
    "name": "Arnsdorf, Stolpener Str. 2",
    "brand": "HEM",
    "street": "Stolpener Str.",
    "house_number": "2",
    "post_code": 1477,
    "place": "Arnsdorf",
    "lat": 51.097607,
    "lng": 13.987778,
    "isOpen": true
  },

**Comment by fewieden** [comment]
@MacG-DD @leovandavart I am currently working on the next version of the module which also contains a new config option `showBrand`.

Either you wait until it is released or you can test the beta version on the develop branch.

```
cd ~/MagicMirror/modules/MMM-Fuel
git checkout develop
git pull
npm i --production
```

**Comment by fewieden** [comment]
@MacG-DD @leovandavart it's now released in version 2.3.0 on master
